ext/AbstractDifferentiationZygoteExt.jl

```diff
@@ -8,11 +8,27 @@ else
     using ..Zygote: Zygote
 end
 
-AD.ZygoteBackend() = AD.ReverseRuleConfigBackend(Zygote.ZygoteRuleConfig())
-
 # Context should not persist between different AD calls: fixes #69
 function AD.ruleconfig(::AD.ReverseRuleConfigBackend{<:Zygote.ZygoteRuleConfig})
     return Zygote.ZygoteRuleConfig()
 end
 
+function AD.value_and_pullback_function(::AD.ZygoteBackend, f, args...)
+    return Zygote.pullback(f, args...)
+end
+
+AD.gradient(::AD.ZygoteBackend, f, args...) = Zygote.gradient(f, args...)
+function AD.value_and_gradient(::AD.ZygoteBackend, f, args...)
+    res = Zygote.withgradient(f, args...)
+    return res.val, res.grad
+end
+
+AD.jacobian(::AD.ZygoteBackend, f, args...) = Zygote.jacobian(f, args...)
```

Comment:
can we provide a way to opt out of this, perhaps a type parameter?

Reply:
We don't provide any opt-outs for any other backend, so my feeling is that we should not add one here. I think it would also unnecessarily complicate the backend (and if there's a problem with `Zygote.jacobian` I still think it should be fixed in Zygote). If a user really wants to use a different implementation of `AD.jacobian` I think it should be reasonably easy to define a custom `MyZygoteBackend` that forwards everything but `AD.jacobian` to a `ZygoteBackend`.

Reply:
I might be the minority vote here in which case I will take the extra inconvenience of defining my own backend.

Reply:
I agree with David on this one